frontend/src/components/common/authed-menu/authed-menu.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC } from 'common/types/types';
+import { getValidClasses } from 'helpers/helpers';
+import { useLocation } from 'hooks/hooks';
+import { Link } from 'components/common/common';
+import { routes } from './common';
+import styles from './styles.module.scss';
+
+const AuthedMenu: FC = () => {
```

Comment:
```suggestion
const AuthorizedWrapper: FC = () => {
```

Reply:
`AuthorizedWrapper` already exists, did you mean `AuthorizedMenu`?